app/views/api/v0/classified_listings/_listing.json.jbuilder

```diff
@@ -0,0 +1,11 @@
+json.type_of              "classified_listing"
+json.id                   listing.id
+json.title                listing.title
+json.slug                 listing.slug
+json.body_markdown        listing.body_markdown
+json.cached_tag_list      listing.cached_tag_list
+json.category             listing.category
+json.processed_html       listing.processed_html
+json.published            listing.published
+json.slug                 listing.slug
+json.tag_list             listing.tag_list
```

Comment:
I just noticed this, sorry if I didn't catch it in the earlier review. 

Let's use the same convention we use in articles:

```ruby
json.tag_list           listing.cached_tag_list
json.tags               listing.tag_list
```

Reply:
I have done so, as well as a few other changes.